README.md

```diff
@@ -1,2 +1,37 @@
 # credentials-io
 Library for reading/writing credentials
+
+## API
+
+### verify(identity, options, callback)
+Verifies digital signature on identity object and any credentials it contains.
+The callback recieves the following object:
+```
+{
+ verified: <true/false>,
+ identity: <framed identity>,
+ domain: <domain the signature was signed for>,
+ publicKey: <public key object for the creator of the signature>,
+ publicKeyOwner: <identity object for the creator of the signature>,
+ // a map of verified objects for each credential (ID => verified)
+ credentials: {}
+}
+```
+
+### send(options, callback)
+Sends credentials to a URL using http-signatures for authentication.
+Options parameter should include following properties:
+```
+{
+  url,
+  credential,
+  privateKeyPem,
+  publickKeyId
+}
+```
+
+### checkDomain(domain, options, callback)
```

Comment:
If this call isn't a part of the public API, which I don't think it is, then there is no need to document it (although, we may want to rename it to _checkDomain). You'll want to talk to @dlongley about this.